### PR TITLE
updated project get to return meta, also granted and draft

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: lib/routers/establishment/project-versions.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const { Router } = require('express');
 const isUUID = require('uuid-validate');
 const { permissions } = require('../../middleware');
-const { NotFoundError } = require('../../errors');
+const { NotFoundError, BadRequestError } = require('../../errors');
+
+const perms = task => permissions(task, (req, res) => ({ licenceHolderId: req.project.licenceHolderId }));
```
Comment (lennym) [inline lib/routers/establishment/project-versions.js]: I'm mildly surprised that lint hasn't 🐄-ed about the unused `res` here.

Comment (joefitter) [reply lib/routers/establishment/project-versions.js]: hmm me too

File: lib/routers/establishment/projects.js [hunk 2]
```diff
@@ -85,41 +91,41 @@ router.param('id', (req, res, next, id) => {
     .then(() => Project.query().findById(id).where('establishmentId', req.establishment.id))
     .then(project => {
       return ProjectVersion.query()
```
Comment (lennym) [inline lib/routers/establishment/projects.js]: We _probably_ want an `if (!project) throw new NotFoundError()` here. Unless that gets caught further up the stack?

Comment (joefitter) [reply lib/routers/establishment/projects.js]: right you are

**Pull request merged**